### PR TITLE
[DOC] Add index page

### PR DESCRIPTION
**Pull request by m273d15**
Implementation of #167

### CODE REVIEW

File: docs/_includes/index_navbar.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ul class="list-group">
+    <a class="list-group-item list-group-item-action flex-column align-items-start active bg-blue">
```
Comment (tobous) [inline docs/_includes/index_navbar.html]: I would prefer another color for the "Resources" header box. From the used blue I would infer that the box is interactive/can be clicked or is a currently selected tab.

Comment (m273d15) [reply docs/_includes/index_navbar.html]: Do you have any idea for alternatives? I create two other versions:

**CURRENT VERSION**
index

**ALTERNATIVE VERSION 1**
* Resources entry has the same color as the navigation bar of the other sites
res1

**ALTERNATIVE VERSION 2**
* Removed the resources entry. Used light blue for list entries and made jumbotron white
* If hover over a list entry, the background of the entry becomes white
res2

Comment (tobous) [reply docs/_includes/index_navbar.html]: Both look fine to me. I would prefer alternative 2, but that is just a matter of personal preference.

Comment (chrismotian) [reply docs/_includes/index_navbar.html]: I like the second one more, too.

Comment (m273d15) [reply docs/_includes/index_navbar.html]: Implemented "Alternative 2"

**Pull request merged**